lms/djangoapps/certificates/models.py

```diff
@@ -945,6 +953,24 @@ def set_language_specific_templates_enabled_for_course(cls, course_key, is_enabl
             defaults=default
         )
 
+    @classmethod
+    def is_hours_of_effort_included_for_course(cls, course_key, ):
```

Comment:
I think that trailing comma isn't supposed to be there.

Reply:
you are correct it is not